File: truth_set/python/visual-representation/bracket-usage/brackets.py
def apply_discount(is_member, discount_available, purchase_amount):
    if is_member and discount_available or purchase_amount > 100:
        print("Discount applied.")

def calculate_total_price(base_price, tax_rate, discount):
    total = base_price + tax_rate * discount  
    print(f"Total price is: {total}")

File: truth_set/go/visual-representation/bracket-usage/brackets.go
package main

import "fmt"

func processOrder(isValidOrder bool) {
	if isValidOrder {
		fmt.Println("Processing order...")
    }
	fmt.Println("Order processed successfully.")
}

type User struct {
	isActive bool
	isAdmin  bool
}

func authenticateUser(user User) {
	if user.isActive {
		if user.isAdmin {
			fmt.Println("Granting admin privileges.")
		} else {
			fmt.Println("Denying access.")
		}
	}
}

func applyDiscount(isMember bool, discountAvailable bool, purchaseAmount float64) {
	if isMember && discountAvailable || purchaseAmount > 100 {
		fmt.Println("Discount applied.")
	}
}

func calculateTotalPrice(basePrice, taxRate, discount float64) {
	total := basePrice + taxRate * discount
	fmt.Printf("Total price is: %f\n", total)
}
\ No newline at end of file